src/cloudformation_cli_python_lib/recast.py

```diff
@@ -127,5 +127,5 @@ def get_forward_ref_type() -> Any:
     # introspection is valid:
     # https://docs.python.org/3/library/typing.html#typing.ForwardRef
     if "ForwardRef" in dir(typing):
-        return typing.ForwardRef  # type: ignore
+        return typing.ForwardRef
```

Comment:
removed "# type: ignore" because of mypy complaining of `unused 'type: ignore' comment `